libraries/AP_Common/AP_Common.h

```diff
@@ -77,6 +77,8 @@
   bool. Bitnumber starts at 0 for the first bit
  */
 #define BIT_IS_SET(value, bitnumber) (((value) & (1U<<(bitnumber))) != 0)
+#define BIT_SET(value, bitnumber) ((value) |= (((typeof(value))1U) << (bitnumber)))
```

Comment:
Great to see BIT_SET and BIT_CLEAR added!

Reply:
I was surprised we did not have any!